flasc/utilities/tuner_utilities.py

```diff
@@ -49,6 +49,10 @@ def replicate_nan_values(
     # Identify common columns between df_1 and df_2
     common_columns = df_1.columns.intersection(df_2.columns)
 
+    # Remove the time column from the common columns if included
+    if "time" in common_columns:
+        common_columns = common_columns.drop("time")
```

Comment:
Might be cleaner to use pandas' built-in check for whether the column exists to avoid the `if` statement?
```pandas
common_columns.drop("time", errors="ignore")
```
? 

Either way is fine though.

Reply:
Good suggestion, using that syntax, thanks!